src/components/Create/components/pages/ProjectToken/components/CustomTokenSettings/components/ReservedTokenRateCallout/ReservedTokenRateCallout.tsx

```diff
@@ -25,12 +25,20 @@ export const ReservedTokenRateCallout: React.FC = () => {
   }, [initialMintRate, reservedTokens])
 
   return (
-    <Callout>
-      <Trans>
-        Contributor rate: {formattedNum(contributorTokens)} / 1 ETH
-        <br />
-        Reserved rate: {formattedNum(reservedTokens)} / 1 ETH
-      </Trans>
-    </Callout>
+    <CreateCallout.Info noIcon collapsible={false}>
+      <span>
+        <Trans>Contributor rate:</Trans>
```

Comment:
**suggestion** still good to have the `Trans` encapsulate the full `key: value`. In some languages it reads `value: key`.